src/ansys/motorcad/core/methods/rpc_methods_general.py

```diff
@@ -356,31 +357,45 @@ def save_to_file(self, mot_file):
         return self.connection.send_and_receive(method, params)
 
     def save_results(self, solution_type):
-        """Save the output results from an ``"EMagnetic"`` solution.
+        """Save the output results from an ``"EMagnetic" or "Thermal"`` solution.
 
-        This method supports only ``"EMagnetic"`` solutions.
+        This method supports only ``"EMagnetic" or "Thermal"`` solutions.
 
         Parameters
         ----------
         solution_type : str
-            Soultion type, which must be ``"EMagnetic"``.
+            Soultion type, which must be ``"EMagnetic" or "Thermal"``.
         """
         method = "SaveResults"
         params = [solution_type]
+
+        if solution_type == "Thermal":
+            self.connection.ensure_version_at_least("2025.0")
+        elif solution_type != "EMagnetic":
+            raise MotorCADError(
+                "SaveResults not available for this solution type: " + solution_type
```

Comment:
Looks good thanks. Maybe worth adding the available types here too.

Reply:
added new line to say available types